src/renderer/components/+extensions/extensions.tsx

```diff
@@ -62,6 +62,9 @@ export class Extensions extends React.Component {
   @observable search = "";
   @observable installPath = "";
 
+  // True if the preliminary install steps have started, but unpackExtension has not started yet
+  @observable startingInstall = false;
```

Comment:
```suggestion
  @observable installing = false;
```

Reply:
"installing" would be misleading, as it's only true in the initial phase of the install. There is a computed `isInstalling` which returns e.g. "installing".